src/middlewared/middlewared/plugins/activedirectory.py

```diff
@@ -1873,8 +1873,8 @@ def parse_msg(self, data):
         self.state = new_state
 
     def read_messages(self):
-        while not self.finished.is_set():
-            with open('/var/run/samba4/.wb_fifo') as f:
+        with open('/var/run/samba4/.wb_fifo') as f:
```

Comment:
Can the fifo be closed if samba is restarted and we need to keep retrying still? @anodos325